doc/radosgw/layout.rst

```diff
@@ -36,15 +50,22 @@ You can use the following commands to introspect metadata entries: ::
     $ radosgw-admin metadata get bucket.instance:<bucket>:<bucket_id>
     $ radosgw-admin metadata get user:<user>   # get or set
     
-Some variables have been used in above commands, they are:
+Variables are used in above command examples; when issuing commands you must
+substitute your specific values:
 
+<<<<<<< HEAD
 - user: Holds user information
 - bucket: Holds a mapping between bucket name and bucket instance id
 - bucket.instance: Holds bucket instance information[2]
+=======
+- ``user``: Holds user information
+- ``bucket``: Holds a mapping between bucket name and bucket instance id
+- ``bucket.instance``: Holds bucket instance information[2]
+>>>>>>> a7dcc3cf242 (doc/radosgw: Improve formatting in layout.rst)
```

Comment:
There seems to be an unresolved conflict in the diff, 

```suggestion
- ``user``: Holds user information
- ``bucket``: Holds a mapping between bucket name and bucket instance id
- ``bucket.instance``: Holds bucket instance information[2]
```

Reply:
I'll take care of it in the morning. Good catch.